resources/views/account/requestable-assets.blade.php

```diff
@@ -109,8 +109,14 @@ class="table table-striped snipe-table"
 
                                                 </td>
 
+                                                <td>
+                                                    @if (Gate::allows('superadmin'))
```

Comment:
Should this be superadmin, or just the ability to view asset models? 

`  @can('view', \App\Models\AssetModel::class)`

Reply:
I think that's a lot better, yes.